users/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from .managers import CustomUserManager
+
+
+class CustomUser(AbstractBaseUser, PermissionsMixin):
+    username = models.EmailField(_("email address"), unique=True)
+    email = models.EmailField(_("email address"), unique=True)
+    first_name = models.CharField(_("first name"), max_length=100)
+    last_name = models.CharField(_("last name"), max_length=100)
+    is_staff = models.BooleanField(default=False)
```

Comment:
I don't think this makes sense for us - users will be staff by definition

Reply:
`is_staff` is a value that is used to determine if someone can log in to the Django admin panel.